iot-e2e-tests/common/src/test/java/tests/integration/com/microsoft/azure/sdk/iot/digitaltwin/DigitalTwinClientTests.java

```diff
@@ -90,7 +92,7 @@ public void cleanUp() {
 
     private DeviceClient createDeviceClient(IotHubClientProtocol protocol) throws IOException, IotHubException, URISyntaxException {
         ClientOptions options = new ClientOptions();
-        options.setModelId(E2ETestConstants.MODEL_ID);
+        options.setModelId(E2ETestConstants.THERMOSTAT_MODEL_ID);
```

Comment:
We should call it a TEST_MODEL_ID instead

Reply:
We have two different modelIds. I used these terms to differentiate.